dotcom-rendering/package.json

```diff
@@ -48,8 +48,8 @@
 		"@guardian/bridget": "2.6.0",
 		"@guardian/browserslist-config": "5.0.0",
 		"@guardian/cdk": "50.13.0",
-		"@guardian/commercial": "13.0.0",
-		"@guardian/consent-management-platform": "13.7.1",
+		"@guardian/commercial": "0.0.0-beta-20240129163326",
```

Comment:
has this been checked with @guardian/commercial-dev ?

Reply:
I would recomment keeping your PR in draft mode whilst it is not ready for review:

<img width="447" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/76776/1b7c32d7-0e0a-4031-8403-84bf02763bed">